stable/stolon/templates/keeper-statefulset.yaml

```diff
@@ -52,7 +52,8 @@ spec:
               IFS='-' read -ra ADDR <<< "$(hostname)"
               export STKEEPER_UID="{{ .Values.keeper.uid_prefix }}${ADDR[-1]}"
               export POD_IP=$(hostname -i)
-              export STKEEPER_PG_LISTEN_ADDRESS=$POD_IP
+              export STKEEPER_PG_ADVERTISE_ADDRESS=$POD_IP
+              export STKEEPER_PG_LISTEN_ADDRESS=${STKEEPER_PG_LISTEN_ADDRESS:-$POD_IP}
```

Comment:
I don't see a way to set this value through the chart.
Is it supposed to be set using `keeper.extraEnv` map?
if so, can we add some comment and/or example?

Reply:
Yes, one would need to set it using ``keeper.extraEnv``. There is already a link in the chart [README](https://github.com/sorintlab/stolon/blob/master/doc/commands_invocation.md#commands-invocation) which lists all the command line options to keeper.
>Environment variables are prefixed with a string related to the command and take the name of the options but are UPPERCASE and any dashes are replaced by underscores.